tests/Doctrine/Tests/Annotations/AbstractReaderTest.php

```diff
@@ -403,9 +403,22 @@ public function testInvalidAnnotationButIgnored()
         $class  = new \ReflectionClass(Fixtures\ClassDDC1660::class);
 
         self::assertTrue(class_exists(Fixtures\Annotation\Version::class));
-        self::assertEmpty($reader->getClassAnnotations($class));
```

Comment:
This change highlights a major BC break: if the patch cannot be applied without a change of an existing test, then it should likely target the next major release of the library.

Reply:
This patch can be definitely applied without this change, but with a separate test. Just give me a moment.

Reply:
Thanks!

Reply:
OK, done. Should I squash commits into one? Else a squash merge is fine by me.

Reply:
If possible, please squash, yes :-)

Reply:
Done.